packages/markup-frontstage/src/MarkupFrontstage.ts

```diff
@@ -0,0 +1,77 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @module Common */
```

Comment:
do you need this line? `/** @module Common */`

Reply:
removed please :)